Bottom: 4688e7075248cb5201dc2a166efc0483f4d0001a
Top:    46989ae0d887cd6613928e3ecf8cfdfbce44b9ce
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:30 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 7039457..55571cf 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1251,8 +1251,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                                 copper_appearance,
                                 NULL));
-
-      //break;
     }
